### DISCUSSION TITLE
E possivel resolver travamentos

### DISCUSSION THREAD

**Discussion by sawrhiyt86** [TeamVanced/VancedManager#789] [General]
E so você ir nas configurações do seu celula vai em amarzenamento interno e vai no cartão e vai nos tres pontos e vais configurações e vai em formatar como portátil

**Comment by cortez150** [comment]
Then what

**Reply by sawrhiyt86** [reply]
And just go to the settings of your cell phone, go to internal storage and go to the card and go to the three points and go to settings and go to format as portable

**Comment by cortez150** [comment]
no puedo youtube funciona es el inicio de sesión de la cuenta que no
funciona no puedo agregar mi cuenta a él
